### PR TITLE
[Merged by Bors] - chore: remove some unnecessary `have`s

### PR DESCRIPTION
**Pull request by mo271**
drive-by: convert some `convert` into `exact`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
!bench

File: Mathlib/Analysis/Calculus/ContDiff/Basic.lean [hunk 1]
```diff
@@ -1070,23 +1070,22 @@ theorem hasFTaylorSeriesUpToOn_pi :
...
+    exact (L m).continuous.comp_continuousOn <| continuousOn_pi.2 fun i => (h i).cont m hm
 
 @[simp]
 theorem hasFTaylorSeriesUpToOn_pi' :
     HasFTaylorSeriesUpToOn n Φ P' s ↔
       ∀ i, HasFTaylorSeriesUpToOn n (fun x => Φ x i)
         (fun x m => (@ContinuousLinearMap.proj 𝕜 _ ι F' _ _ _ i).compContinuousMultilinearMap
           (P' x m)) s := by
-  convert hasFTaylorSeriesUpToOn_pi (𝕜 := 𝕜) (φ := fun i x ↦ Φ x i); ext; rfl
+    convert hasFTaylorSeriesUpToOn_pi (𝕜 := 𝕜) (φ := fun i x ↦ Φ x i); ext; rfl
```
Comment (vihdzp) [inline Mathlib/Analysis/Calculus/ContDiff/Basic.lean:1088]: Isn't the usual convention to indent proofs by two spaces?

**Review by vihdzp — APPROVED** [review]
Looks like an improvement to me.

File: Mathlib/Analysis/Calculus/ContDiff/Basic.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/Analysis/Calculus/ContDiff/Basic.lean]: good catch. (I hope auto-formatting is coming soon...)

**Review by Ruben-VandeVelde — APPROVED** [review]
I'm not sure all of those are necessarily big improvements, but they're at least not worse, and some are better.

maintainer merge

**Comment by urkud** [comment]
Some of these `have`s and `convert`s were there to speed up compilation by changing the order of elaboration. Not sure if this is still relevant with current Lean.
!bench

**Comment by urkud** [comment]
If `!bench` succeeds and doesn't report slowdowns, then please go ahead and merge this PR. Thanks!
bors d+

**Comment by semorrison** [comment]
!bench

(trying again, it failed previously)

**Comment by semorrison** [comment]
!bench

**Pull request closed without merging**